front/pages/w/[wId]/members/index.tsx

```diff
@@ -262,12 +275,9 @@ function WorkspaceMembersList({
     searchTerm,
     pageIndex: pagination.pageIndex,
     pageSize: DEFAULT_PAGE_SIZE,
+    groupKind: isProvisioningEnabled ? "provisioned" : undefined,
   });
 
-  useEffect(() => {
```

Comment:
There are two useEffect and I think they are useless since setPagination is a stable reference??

Reply:
I think I'm confused, but when we mount, we initialize with this, so no need to reset? 
```pageIndex: 0,
    pageSize: DEFAULT_PAGE_SIZE,```

but I will bring one back so I don't break anything for demo 😄

Reply:
actually they were in different components, I will bring back both